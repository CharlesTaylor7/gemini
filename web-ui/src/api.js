export const fetchRovers = () => fetch('/rovers').then((res) => res.json());

export const fetchPhotos = ({ rover, camera }) => fetch(
  `/photos?rover=${rover}&camera=${camera}`,
).then((res) => res.json());
