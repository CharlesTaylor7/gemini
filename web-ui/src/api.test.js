import { fetchPhotos } from './api';

describe('fetchPhotos', () => {
  test('integration test', async () => {
    const [photos1, photos2] = await Promise.all([
      fetchPhotos({
        rover: 'Curiosity',
        camera: 'FHAZ',
        method: 'SQL',
      }),
      fetchPhotos({
        rover: 'Curiosity',
        camera: 'FHAZ',
        method: 'ORM',
      }),
    ]);
    expect(photos1).toMatchSnapshot();
    expect(photos1).toEqual(photos2);
  });
});
