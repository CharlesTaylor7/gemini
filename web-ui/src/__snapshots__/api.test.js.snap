// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchPhotos integration test 1`] = `
Array [
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FRA_397506068EDR_D0010008AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FRA_397502305EDR_D0010000AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FLA_397506068EDR_D0010008AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FRA_397506083EDR_F0010008AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FLA_397504830EDR_F0010000AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FLA_397506083EDR_F0010008AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FLA_397502305EDR_D0010000AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-06",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00000/opgs/edr/fcam/FRA_397504830EDR_F0010000AUT_04096M_.JPG",
  },
  Object {
    "date": "2012-08-20",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00014/opgs/edr/fcam/FRA_398741806EDR_F0030004FHAZ00200M_.JPG",
  },
  Object {
    "date": "2012-08-20",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00014/opgs/edr/fcam/FRA_398742695EDR_F0030004FHAZ00200M_.JPG",
  },
  Object {
    "date": "2012-08-20",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00014/opgs/edr/fcam/FLA_398742695EDR_F0030004FHAZ00200M_.JPG",
  },
  Object {
    "date": "2012-08-20",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00014/opgs/edr/fcam/FLA_398741806EDR_F0030004FHAZ00200M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398830978EDR_S0030008FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398831049EDR_S0030016FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398830957EDR_S0030008FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398831028EDR_S0030016FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398831028EDR_S0030016FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398831049EDR_S0030016FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398830957EDR_S0030008FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398831344EDR_F0030044FHAZ00103M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398831344EDR_F0030044FHAZ00103M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398830978EDR_S0030008FHAZ00102M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FLA_398830747EDR_F0030004FHAZ00105M_.JPG",
  },
  Object {
    "date": "2012-08-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00015/opgs/edr/fcam/FRA_398830747EDR_F0030004FHAZ00105M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FRA_398917334EDR_F0030060FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FRA_398917511EDR_F0030066FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FLA_398917334EDR_F0030060FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FRA_398919417EDR_F0030078FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FLA_398919417EDR_F0030078FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FLA_398917511EDR_F0030066FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FLA_398917092EDR_F0030054FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-22",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00016/opgs/edr/fcam/FRA_398917092EDR_F0030054FHAZ00130M_.JPG",
  },
  Object {
    "date": "2012-08-27",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00021/opgs/edr/fcam/FRA_399365049EDR_F0030100FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-27",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00021/opgs/edr/fcam/FLA_399365049EDR_F0030100FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-28",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00022/opgs/edr/fcam/FLA_399455394EDR_F0030260FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-28",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00022/opgs/edr/fcam/FRA_399455394EDR_F0030260FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399620448EDR_D0030266TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399621579EDR_D0030308TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399620448EDR_D0030266TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399620752EDR_D0030284TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399620752EDR_D0030284TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399620930EDR_D0030290TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399621116EDR_D0030296TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399621116EDR_D0030296TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399621298EDR_D0030302TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399621579EDR_D0030308TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399621298EDR_D0030302TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399621772EDR_D0030314TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399621772EDR_D0030314TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399621964EDR_D0030320TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399621964EDR_D0030320TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399622157EDR_D0030326TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399622157EDR_D0030326TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399620930EDR_D0030290TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399622385EDR_D0030332TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399622385EDR_D0030332TRAV00004M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FLA_399627431EDR_F0030372FHAZ00303M_.JPG",
  },
  Object {
    "date": "2012-08-30",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00024/opgs/edr/fcam/FRA_399627431EDR_F0030372FHAZ00303M_.JPG",
  },
  Object {
    "date": "2012-09-01",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00026/opgs/edr/fcam/FLA_399806994EDR_F0030530FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-01",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00026/opgs/edr/fcam/FRA_399806994EDR_F0030530FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-04",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00029/opgs/edr/fcam/FRA_400070892EDR_F0040000FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-04",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00029/opgs/edr/fcam/FLA_400070892EDR_F0040000FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-04",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00029/opgs/edr/fcam/FRA_400066496EDR_F0030872FHAZ00301M_.JPG",
  },
  Object {
    "date": "2012-09-04",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00029/opgs/edr/fcam/FLA_400066496EDR_F0030872FHAZ00301M_.JPG",
  },
  Object {
    "date": "2012-09-05",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00030/opgs/edr/fcam/FRA_400162288EDR_F0040000FHAZ00201M_.JPG",
  },
  Object {
    "date": "2012-09-05",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00030/opgs/edr/fcam/FLA_400162288EDR_F0040000FHAZ00201M_.JPG",
  },
  Object {
    "date": "2012-09-05",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00030/opgs/edr/fcam/FRA_400161925EDR_F0040000FHAZ00201M_.JPG",
  },
  Object {
    "date": "2012-09-05",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00030/opgs/edr/fcam/FLA_400161925EDR_F0040000FHAZ00201M_.JPG",
  },
  Object {
    "date": "2012-09-13",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00037/opgs/edr/fcam/FLA_400781141EDR_F0040000FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-13",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00037/opgs/edr/fcam/FRA_400781141EDR_F0040000FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-14",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00038/opgs/edr/fcam/FRA_400871396EDR_F0040404FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-14",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00038/opgs/edr/fcam/FLA_400871396EDR_F0040404FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-15",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00039/opgs/edr/fcam/FLA_400958554EDR_F0040468FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-15",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00039/opgs/edr/fcam/FRA_400958554EDR_F0040468FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-16",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00040/opgs/edr/fcam/FLA_401051057EDR_F0040916FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-16",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00040/opgs/edr/fcam/FRA_401051057EDR_F0040916FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-17",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00041/opgs/edr/fcam/FRA_401137864EDR_F0041238FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-17",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00041/opgs/edr/fcam/FLA_401137864EDR_F0041238FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-18",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00042/opgs/edr/fcam/FRA_401232778EDR_F0041632FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-18",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00042/opgs/edr/fcam/FLA_401232778EDR_F0041632FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-19",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00043/opgs/edr/fcam/FLA_401316557EDR_F0042002FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-19",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00043/opgs/edr/fcam/FRA_401316557EDR_F0042002FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490139EDR_D0042080TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490139EDR_D0042080TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490451EDR_D0042086TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490712EDR_D0042092TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490712EDR_D0042092TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490207EDR_F0042080FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490207EDR_F0042080FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490495EDR_F0042086FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490495EDR_F0042086FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490757EDR_F0042092FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401490757EDR_F0042092FHAZ00306M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401490451EDR_D0042086TRAV00014M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FLA_401492677EDR_F0042100FHAZ00103M_.JPG",
  },
  Object {
    "date": "2012-09-21",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00045/opgs/edr/fcam/FRA_401492677EDR_F0042100FHAZ00103M_.JPG",
  },
  Object {
    "date": "2012-09-24",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00048/opgs/edr/fcam/FLA_401761250EDR_F0042644FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-24",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00048/opgs/edr/fcam/FRA_401761250EDR_F0042644FHAZ00304M_.JPG",
  },
  Object {
    "date": "2012-09-25",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00049/opgs/edr/fcam/FLA_401848200EDR_F0042778FHAZ00302M_.JPG",
  },
  Object {
    "date": "2012-09-25",
    "url": "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00049/opgs/edr/fcam/FRA_401848200EDR_F0042778FHAZ00302M_.JPG",
  },
]
`;
